src/components/pages/Layout/index.tsx

```diff
@@ -8,12 +9,16 @@ import useCartContext from 'contexts/CartContext/useCartContext'
 
 const Layout = () => {
   const { isOpen } = useCartContext()
+
+  const classes = classNames(
+    'flex min-h-screen w-full flex-col justify-between',
+    {
+      'fixed h-screen': isOpen,
+    },
+  )
+
   return (
-    <div
-      className={`${
-        isOpen ? 'fixed h-screen' : ''
-      }flex min-h-screen w-full flex-col justify-between`}
-    >
+    <div className={classes}>
```

Comment:
I'd rather like you to have classes here as far as these are not too large, then we would want to keep them in `@apply`